modules/local/ms2rescore.nf

```diff
@@ -0,0 +1,55 @@
+process MS2RESCORE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::ms2rescore=3.0.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ms2rescore:3.0.1--pyhdfd78af_2':
+        'biocontainers/ms2rescore:3.0.1--pyhdfd78af_2' }"
+
+    // userEmulation settings when docker is specified
+    containerOptions = (workflow.containerEngine == 'docker') ? '-u $(id -u) -e "HOME=${HOME}" -v /etc/passwd:/etc/passwd:ro -v /etc/shadow:/etc/shadow:ro -v /etc/group:/etc/group:ro -v $HOME:$HOME' : ''
```

Comment:
would it be better to add this to the `modules.config` file? that way it can be overridden if necessary

Reply:
I think it ended up here because `workflow.containerEngine` is not accessible in `conf/modules.config`.